crates/Cargo.toml

```diff
@@ -13,3 +13,5 @@ members = [
 codegen-units = 1
 panic = "abort"
 
+[patch.crates-io]
+sqlite3-parser = { git = "https://github.com/LucioFranco/lemon-rs" }
```

Comment:
That's ok for now, but once we migrate off rusqlite, we'd need to get rid of this one too in order to finally publish `sqld` (or rather `libsql-server` on crates. @MarinPostma I think it's inevitable to fork lemon-rs and make it produce a parser from our libsql/ directory base? We're going to need it anyway to parse our syntax extensions like `CREATE FUNCTION`. Alternatively we can hack the build system in libsql/ to also produce Rust code from parse.y, but that's more or less what lemon-rs does right now, isn't it.

Reply:
Yeah, id like to bring this fork into libsql proper so we don't even need to depend on a 3rd party package.

Reply:
We will need to do this to even publish the rust package anyways.

Reply:
yeah, we'll need to fork it for other reasons as well, let's maybe bring it to the org, and not use a patch